### PR TITLE
lsp-plugins: 1.1.10 -> 1.1.13

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/applications/audio/lsp-plugins/default.nix [hunk 1]
```diff
@@ -5,36 +5,24 @@
...
 
   src = fetchFromGitHub {
     owner = "sadko4u";
     repo = pname;
     rev = "${pname}-${version}";
-    sha256 = "09gmwzh1gq1q2lxn8fc1bpdh02h8vr7r0i040c1nx256wgfsarqb";
+    sha256 = "00mhrr873kgcnqy3q0yi1r5zacfcvz7fqpzsmfhw5d095jm970al";
   };
 
   nativeBuildInputs = [ pkgconfig php expat ];
```
Comment (jtojnar) [inline pkgs/applications/audio/lsp-plugins/default.nix]: expat seems to be no longer used.

**Review by jtojnar — COMMENTED** [review]
Changes: https://github.com/sadko4u/lsp-plugins/blob/lsp-plugins-1.1.13/CHANGELOG.txt#L5-L61

File: pkgs/applications/audio/lsp-plugins/default.nix (hunk 1 above)
Comment (magnetophon) [reply pkgs/applications/audio/lsp-plugins/default.nix]: Good catch!

File: pkgs/applications/audio/lsp-plugins/default.nix [hunk 2]
```diff
@@ -1,40 +1,28 @@
...
     owner = "sadko4u";
     repo = pname;
     rev = "${pname}-${version}";
-    sha256 = "09gmwzh1gq1q2lxn8fc1bpdh02h8vr7r0i040c1nx256wgfsarqb";
+    sha256 = "00mhrr873kgcnqy3q0yi1r5zacfcvz7fqpzsmfhw5d095jm970al";
   };
 
-  nativeBuildInputs = [ pkgconfig php expat ];
+  nativeBuildInputs = [ pkgconfig php ];
   buildInputs = [ jack2Full libsndfile libGLU libGL lv2 cairo ladspaH makeWrapper ];
```
Comment (jtojnar) [inline pkgs/applications/audio/lsp-plugins/default.nix]: Could you also move `makeWrapper` to `nativeBuildInputs`.

File: pkgs/applications/audio/lsp-plugins/default.nix [hunk 3]
```diff
@@ -1,40 +1,28 @@
...
 
   makeFlags = [
-    "BIN_PATH=$(out)/bin"
-    "LIB_PATH=$(out)/lib"
-    "DOC_PATH=$(out)/share/doc"
+    "PREFIX=$(out)"
```
Comment (jtojnar) [inline pkgs/applications/audio/lsp-plugins/default.nix]: Placeholder is preferred to relying on GNUMake interpolation:

```suggestion
    "PREFIX=${placeholder ''out''}"
```

**Comment by jtojnar** [comment]
@GrahamcOfBorg build

File: pkgs/applications/audio/lsp-plugins/default.nix [hunk 4]
```diff
@@ -154,6 +142,5 @@ stdenv.mkDerivation rec {
       homepage = https://lsp-plug.in;
       maintainers = with maintainers; [ magnetophon ];
       license = licenses.gpl2;
-      platforms = [ "x86_64-linux" ];
```
Comment (jtojnar) [inline pkgs/applications/audio/lsp-plugins/default.nix]: Can you use `platforms.linux` if it does not build on Darwin and `platforms.unix` if it does?

File: pkgs/applications/audio/lsp-plugins/default.nix [hunk 5]
```diff
@@ -1,40 +1,28 @@
...
-    substituteInPlace ./include/container/jack/main.h --replace "/usr/lib" "$out/lib"
-    substituteInPlace ./include/container/vst/main.h --replace "/usr/lib" "$out/lib"
-    # for https://github.com/sadko4u/lsp-plugins/issues/7#issuecomment-426561549 :
-    sed -i '/X11__NET_WM_WINDOW_TYPE_DOCK;/d' ./src/ui/ws/x11/X11Window.cpp
-    runHook postPatch
-  '';
-
   doCheck = true;
 
   checkPhase = ''
```
Comment (jtojnar) [inline pkgs/applications/audio/lsp-plugins/default.nix]: `buildFlags` should be a list.

File: pkgs/applications/audio/lsp-plugins/default.nix (hunk 4 above)
Comment (magnetophon) [reply pkgs/applications/audio/lsp-plugins/default.nix]: I don't know if it does. Let's find out!

**Comment by magnetophon** [comment]
@jtojnar Thanks for the feedback, all done!

File: pkgs/applications/audio/lsp-plugins/default.nix (hunk 4 above)
Comment (jtojnar) [reply pkgs/applications/audio/lsp-plugins/default.nix]: Does not seem to build on Darwin https://logs.nix.ci/?key=nixos/nixpkgs.76317&attempt_id=39ed18c1-8ad6-47d2-a68b-e16a5fecab6d

Comment (magnetophon) [reply pkgs/applications/audio/lsp-plugins/default.nix]: OK, changed it to linux.

**Comment by jtojnar** [comment]
Thanks.

**Pull request merged**